packages/fluxible-addons-react/connectToStores.js

```diff
@@ -41,12 +42,16 @@ function createComponent(Component, stores, getStateFromStores, customContextTyp
                 this.context.getStore(Store).removeListener('change', this._onStoreChange);
             }, this);
         },
-        componentWillReceiveProps: function componentWillReceiveProps(nextProps){
-            this.setState(this.getStateFromStores(nextProps));
+        componentDidUpdate: function componentDidUpdate(prevProps, prevState) {
```

Comment:
what kind of subtle changes do we expect to see moving from `componentWillReceiveProps` to `componentDidUpdate`?

Reply:
I would say that we would have mainly a performance decrease. Since updating the state after the component is updated we will have a second call to `render` method. Also from the same React documentation section posted above:

> It would also cause an extra re-rendering which, while not visible to the user, can affect the component performance.

Reply:
But I think we can have the same issue that react-redux has: [stale-props](https://react-redux.js.org/api/hooks#stale-props-and-zombie-children).